recipe/build.sh

```diff
@@ -68,6 +68,62 @@ if [[ "$blas_impl" == "accelerate" ]]; then
         -Wl,-reexport_library,$SRC_DIR/accelerate/liblapacke-netlib.${PKG_VERSION}.dylib
 
     cp libvecLibFort-ng.dylib $SRC_DIR/accelerate/
+
+elif [[ "$blas_impl" == "newaccelerate" ]]; then
+    # New Accelerate libraries have all symbols in BLAS, CBLAS, LAPACK with $NEWLAPACK
+    # name appended to all the symbol names. Therefore we create a library that dispatches
+    # eg: _dgemm -> _dgemm$NEWLAPACK with aliases.txt
+    #
+    # One exception to this is {c,z}dot{u,c}_ symbols which have different signatures.
+    # We use wrap_accelerate.c to fix those.
+    #
+    # For LAPACKE symbols, we use the LAPACKE wrappers from netlib which would call
+    # the LAPACK symbols from Accelerate.
```

Comment:
So IIUC, the only reason we're shipping netlibs `liblapacke.dylib` is for the C-wrappers that call back to the new accelerate? I mean, the library's only 400kb, so this is not a real issue, but just to double-check.

Also, let's avoid conjunctives for factual statements:
```suggestion
    # For LAPACKE symbols, we use the LAPACKE wrappers from netlib which will call
    # the LAPACK symbols from Accelerate.
```

Reply:
> So IIUC, the only reason we're shipping netlibs liblapacke.dylib is for the C-wrappers that call back to the new accelerate? 

the whole purpose of liblapacke.dylib is to call the Fortran lapack library which in this case is the new accelerate framework.